jkube-kit/config/image/src/test/java/org/eclipse/jkube/kit/config/image/build/HealthCheckConfigTest.java

```diff
@@ -62,6 +65,48 @@ void testGoodHealthCheck7() {
                 .validate();
     }
 
+    @Test
+    public void testBadHealthCheck1() {
+        HealthCheckConfiguration healthCheckConfiguration = HealthCheckConfiguration.builder()
+                .mode(HealthCheckMode.none)
+                .interval("2s")
+                .build();
+        assertThatThrownBy(healthCheckConfiguration::validate).
+                isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    public void testBadHealthCheck2() {
+        HealthCheckConfiguration healthCheckConfiguration = HealthCheckConfiguration.builder()
+                .mode(HealthCheckMode.none)
+                .retries(1)
+                .build();
+        assertThatThrownBy(healthCheckConfiguration::validate).
+                isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    public void testBadHealthCheck3() {
+        HealthCheckConfiguration healthCheckConfiguration = HealthCheckConfiguration.builder()
+                .mode(HealthCheckMode.none)
+                .timeout("3s")
+                .build();
+        assertThatThrownBy(healthCheckConfiguration::validate).
+        isInstanceOf(IllegalArgumentException.class);
+
+    }
+
+    @Test
+    public void testBadHealthCheck4() {
+        HealthCheckConfiguration healthCheckConfiguration = HealthCheckConfiguration.builder()
+                .mode(HealthCheckMode.none)
+                .startPeriod("30s")
+                .cmd(Arguments.builder().shell("echo a").build())
+                .build();
+        assertThatThrownBy(healthCheckConfiguration::validate).
+                isInstanceOf(IllegalArgumentException.class);
```

Comment:
Minor, please be consistent with code style you choose while coding. It's better to align with what's already there.
```suggestion
                isInstanceOf(IllegalArgumentException.class);
```
```suggestion
        assertThatThrownBy(healthCheckConfiguration::validate)
                .isInstanceOf(IllegalArgumentException.class);
```